### PR TITLE
Tweak dot color

### PR DESCRIPTION
**Pull request by voltrevo**

### CODE REVIEW

**Review by leahciMic — CHANGES_REQUESTED** [review]
I hate this colour

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var placeDot = function(dot, opt) {
 
 var Dot = function(opt) {
   var dot = document.createElement('div');
-  dot.style.backgroundColor = opt.color || 'blue';
+  dot.style.backgroundColor = opt.color || '#04f';
```
Comment (leahciMic) [inline index.js]: Really should be like orange or something

Comment (voltrevo) [inline index.js]: ok...

**Review by leahciMic — APPROVED** [review]
Actually never mind, blue is cool

**Review by leahciMic — APPROVED** [review]
I approve these changes

File: index.js (hunk 1 above)
Comment (leahciMic) [inline index.js]: All good now

**Review by leahciMic — APPROVED** [review]
All good!

**Review by leahciMic — APPROVED** [review]
Go for it

**Review by leahciMic — APPROVED** [review]

**Pull request closed without merging**